test/es6/rlexe.xml

```diff
@@ -690,7 +690,7 @@
   </test>
   <test>
     <default>
-      <files>proxyenumbug.js</files>
+      <files>proxyenumremoval.js</files>
```

Comment:
no change to existing tests? at least the enumerate trap shouldn't have been called. we will need to cleanup those tests.